eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
```

Comment:
Second proposal: I still think `ezpublish-asset-dump-env` should be deprecated, because it is flawed given it is not taken into account for cache clear, so if you' haven't set `SYMFONY_ENV ` to prod, it will potentially crash as cache for prod haven't been cleared on composer updates.

Thump up: Deprecate, and as I and @bdunogier kind of intend start to adapt documentation for optionally setting `SYMFONY_ENV=prod` when doing install, otherwise prod is assumed.
Thumb down: Find or create another composer config that can be taken into account for all install/update commands *(note: I have no idea how this is supposed to work, overriding all scripts is not really an option, maybe we need to make a composer plugin for this)*